composer/workflows/kubernetes_pod_operator_c2.py

```diff
@@ -75,11 +75,12 @@
         # Name of task you want to run, used to generate Pod ID.
         name="pod-ex-minimum",
         # Entrypoint of the container, if not specified the Docker container's
-        # entrypoint is used. The cmds parameter is templated.
```

Comment:
I can see that cmds parameter is actually templated?
https://github.com/apache/airflow/blob/8c9d251efbd07d4c30a90264364847eb529c90d1/airflow/providers/cncf/kubernetes/operators/pod.py#L248

Reply:
Oh, it's templated in a different sense. I'll revert these changes and clarify in the templates example. Thanks!